aten/src/ATen/native/Blas.cpp

```diff
@@ -67,12 +67,12 @@ TORCH_IMPL_FUNC(addmv_out_cpu)(const Tensor &self, const Tensor &mat, const Tens
     if (betaval == 0.0) {
       result.zero_();
     } else {
-      at::cpu::mul_out(
-          // NOLINTNEXTLINE(cppcoreguidelines-pro-type-const-cast)
-          const_cast<Tensor&>(result),
-          self,
-          at::native::scalar_tensor(
-              beta_, self.scalar_type(), c10::nullopt /* layout */, at::kCPU, c10::nullopt /* pin_memory */));
+      // at::cpu::mul_out(
```

Comment:
yep, `at::cpu::mul` and friends no longer work - they are generated by the codegen.

In fact, `at::cpu::mul` is actually a generated function that's meant to skip the dispatcher completely, pretty similar to what your change for `at::mul`.